test/test_list.py

```diff
@@ -130,7 +130,9 @@ def test_list_hst_mappings(capsys):
     assert out_to_check in out
 
 
-def test_list_jwst_mappings(capsys):
+@mark.list
+def test_list_jwst_mappings(capsys, jwst_shared_cache_state):
+    jwst_shared_cache_state.cleanup()
```

Comment:
cleanup should happen only after the commands/functionality being tested. Putting it as the first command defeats the purpose of using the fixture (resets the config state)

Reply:
Thanks, done.